cmd/cmd.go

```diff
@@ -617,7 +618,15 @@ func PushHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	p.Stop()
 	spinner.Stop()
+
+	destination := n.String()
+	if isOllamaHost {
+		destination = "https://ollama.com/" + strings.TrimSuffix(n.DisplayShortest(), ":latest")
+	}
+	fmt.Printf("\nModel pushed → %s\n", destination)
```

Comment:
The `→` is nice, but I wonder if will render correctly in all terminal fonts (similar to the bar edges in `ollama pull` and `ollama push`. Should we perhaps use a colon or similar`:`?